### PR TITLE
Faucet

### PR DESCRIPTION
**Pull request by Maan2003**
motivation: when you don't cli access. 
example: deployed testing federation, wasm test (so wasm can pay and create invoices)
`curl $FM_FAUCET_BIND_ADDR/invoice -d 21`
`curl $FM_FAUCET_BIND_ADDR/pay -d ln......`

### CODE REVIEW

File: devimint/src/vars.rs [hunk 1]
```diff
@@ -104,6 +105,7 @@ declare_vars! {
 
         FM_CLN_EXTENSION_LISTEN_ADDRESS: String = "0.0.0.0:8177";
         FM_GATEWAY_LIGHTNING_ADDR: String = "http://localhost:8177";
+        FM_FAUCET_BIND_ADDR: String = "0.0.0.0:15243";
```
Comment (Maan2003) [inline devimint/src/vars.rs:108]: port number generated by mashing buttons.

**Review by dpc — APPROVED** [review]
:fire: 

Haven't tested, but LGTM, and if there are issue, we can follow up with fixes.

File: devimint/src/vars.rs (hunk 1 above)
Comment (dpc) [reply devimint/src/vars.rs:108]: We might go with something more memorable if we expect users to want to type it in their browser. `19999` or something?

**Comment by justinmoon** [comment]
In particular, this will help us write WASM tests that run in a headless browser using a tool like [wasm-bindgen-test](https://rustwasm.github.io/wasm-bindgen/wasm-bindgen-test/index.html)

**Review by dpc — APPROVED** [review]

**Pull request merged**